lib/index.config.js

```diff
@@ -23,36 +23,20 @@ export default function routes($ocLazyLoadProvider, $stateProvider, $translatePr
 		url: '/page1',
 		views: {
 			mainView: {
-				templateProvider: ($q) => {
-					return $q((resolve) => {
-						// lazy load the view
-						require.ensure([], () => resolve(require("./components/page1/page1.html")));
-					});
-				},
-				controller: function () {
-					this.name = 'Hello Page1';
-				}
+				component: 'page1'
```

Comment:
No need to use template provider.  Simply specify the name of the component, and make sure that the component is registered before the state activates